src/cogitate.py

```diff
@@ -1,58 +1,230 @@
-"""Command Line Interface for interacting with Github repository info."""
+"""Command Line Interface for the Cogitate tool."""
```

Comment:
I suggest the adding of a command that would clear the json files before/after the program is finished running. You can do that by writing an empty dictionary `{}` to the files. This would allow the user to either preserve the current data or to start over. Let me know what you think about this one.

Reply:
I agree this would be a good feature to have. 

I think having this addition as an argument the user can pass to the CLI, with a default to overwriting the json, which will allow the data currently in the json file (if there is any) to be used is the best way we could implement this. 

Doing it another way, will add a call and response with the user that the customer has identified as something to avoid in the program whenever possible as he wishes to be able to run the program, and then just hit up in the terminal, change and argument or two and run the program again. 

If we implement the method I mentioned above, the user has the option, but isn't always forced to respond to a question about it.

Reply:
Since the default would be to overwrite the JSON files, do you plan on adding arguments that delete the contents of each JSON file separately or will both `individual_metrics_storage` and ` raw_data_storage` be cleared. I think it is important to specify that and make it clear. Also, if the user chooses not to overwrite, is the program going to go directly to the percentage of contribution step or will there be a different order. This complicates things a little bit and I just want to make sure everyone agrees on the steps we're taking.

Reply:
I think both storage.json files will be cleared. It doesn't make sense to clear one and not the other. 

I think, and this may be wrong, that if the user chooses not to overwrite, we should still do the calculations off of the raw_data_storage. Because as far as I know there is not currently a  way to correctly retrieve the dict in the Individual_metric_data_storage?  I may be completely wrong let me know xD 
If there is way then yes, we should skip straight to calculating the percentages and printing the tables.

Reply:
I agree that both json files should be cleared at the same time if chosen. If overwrite was not chosen, you can retrieve the individual metrics dictionary using the json handler similar to how you would for the raw data dictionary, just make sure to change the name of the file.